repos.yaml

```diff
@@ -1620,3 +1620,8 @@ repositories:
     maintainers:
       - email: scott@r6by.com
         name: scottrigby
+  - name: drycc
+    url: https://charts.drycc.cc/stable
```

Comment:
Can you add this same information to the file at _config/helm-hub-values.yaml_

Reply:
ok